### PR TITLE
Docker enable

### PR DESCRIPTION
**Pull request by Helvio88**
Hopefully this works. Thanks for the tips!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,52 @@ If you want to build `TinShop` from source, please run `go build`.
 
 And then, simply run `./tinshop`.
 
+# 🐋 Docker
+
+To run with [Docker](https://docs.docker.com/engine/install/), you can use this as a starting `cli` example:
+
+`docker run -d --restart=always -e TINSHOP_SOURCES_DIRECTORIES=/games -e TINSHOP_WELCOMEMESSAGE="Welcome to my Tinshop!" -v /local/game/backups:/games -p 3000:3000`
```
Comment (DblK) [inline README.md:74]: Missing the base image there 😋

```suggestion
`docker run -d --restart=always -e TINSHOP_SOURCES_DIRECTORIES=/games -e TINSHOP_WELCOMEMESSAGE="Welcome to my Tinshop!" -v /local/game/backups:/games -p 3000:3000 ghcr.io/dblk/tinshop:latest`
```

File: README.md [hunk 2]
```diff
@@ -67,6 +67,52 @@ If you want to build `TinShop` from source, please run `go build`.
...
+This will run Tinshop on  `http://localhost:3000` and persist across reboots!
+
+If `docker compose` is your thing, then start with this example:
+
+```yaml
+version: '3.9'
+services:
+  tinshop:
+    container_name: tinshop
+    image: helvio/tinshop:latest
```
Comment (DblK) [inline README.md:85]:
```suggestion
    image: ghcr.io/dblk/tinshop:latest
```

**Comment by Helvio88** [comment]
Should close #44 partially

File: README.md [hunk 3]
```diff
@@ -67,6 +67,52 @@ If you want to build `TinShop` from source, please run `go build`.
 
 And then, simply run `./tinshop`.
 
+# 🐋 Docker
```
Comment (DblK) [inline README.md:70]: In `Use`, please add after `container`:
```mk
(See [`Docker`](https://github.com/DblK/tinshop/tree/master#-docker) section below)
```

PS: I did not test the link so be sure it's working 🙏

**Review by DblK — CHANGES_REQUESTED** [review]
I will need to test the code, meanwhile please update the readme

File: README.md (hunk 3 above)
Comment (Helvio88) [reply README.md:70]: Should work, but, maybe the link shouldn't include `/tree/master`?

Comment (Helvio88) [reply README.md:70]: I see other links include it, so I kept it the same way. If they should change, we'll fix them all at once.

**Pull request merged**